aquacomputer_d5next.c

```diff
@@ -502,17 +502,13 @@ static int aqc_send_ctrl_data(struct aqc_data *priv)
 	return ret;
 }
 
-/* Refreshes the control buffer and stores value at offset in val */
+/* Stores value at offset in buffer in val */
 static int aqc_get_ctrl_val(struct aqc_data *priv, int offset, long *val, size_t size)
 {
 	int ret;
 
 	mutex_lock(&priv->mutex);
 
-	ret = aqc_get_ctrl_data(priv);
```

Comment:
A userspace app may have changed the config (using hidraw) without us knowing, so we must still refresh.

As far as I know the sensor report (for any device) does not contain the temp offset, but I'll check again just to be sure. Guenter asked that because AFAIK devices (usually) send that info along with the sensor value, kind of like an accuracy measurement.